assets/src/styles/blocks/PageHeader.scss

```diff
@@ -4,6 +4,17 @@ $text-column-width-xl: 380px;
 $text-column-width-xxl: 440px;
 $text-column-padding-in: 24px;
 
+.wp-block-group.is-pattern-p4-page-header {
+  .wp-block-media-text__media img {
+    transform: translate3d(0px, 0px, 0px) !important;
```

Comment:
This change removes the parallax effect that was [added to this pattern recently](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/commit/7f9e8261d4b0817015b1d7d90ae5cebbda4d8286), I don't think that's what we want?

Reply:
That doesn't solve the issue because the `.is-style-parallax` class is added to the `wp-block-media-text` element, and not `.wp-block-media-text__media` like it's implemented here 😬

Reply:
I agree but the main point is of this PR is to validate this [way of setting the padding](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/969/files#diff-67932f3eea932974a0145a4fb8a1acc624cae0c611a9627c09d73fa83f1da489R12-R15).

Removing `transform: translate3d(0px, 0px, 0px) !important;` it's only because of a small gap it has but doesn't affect the main issue. I'll remove that part to avoid confusions!

Reply:
Done!